docs/content/en/docs/configuration/command-line.md

```diff
@@ -376,6 +448,27 @@ See also:
 
 ---
 
+## --update-status
+
+Indicates whether the ingress controller should update the Ingress status IP/hostname. Defaults to `true`.
+
+---
+
+## --update-status-on-shutdown
+
+Indicates whether the ingress controller should update the Ingress status IP/hostname when the
+controller is being stopped. Defaults to `true`.
+
+---
+
+## --validate-config
+
+Determines whether the resulting configuration files should be validated when a dynamic update was
+applied. Default value is `false`, which means the validation will only happen when HAProxy needs to
+be reloaded.
```

Comment:
Should i describe the behaviour if a dynamic update causes invalid configuration?  Will it just ignore the update, or crash?

Reply:
Added a few more words in the new output